src/translations/nbTranslations.json

```
@@ -1082,8 +1080,8 @@
       "department": "Enhet",
       "identification": "Identifisering",
       "identity_status": {
-        "confirmed_identity": "Bekreftet identitet",
-        "confirmed_identity_without_affiliation": "Bekreftet identitet uten tilhørighet",
+        "confirmed_identity": "Identifisert",
+        "confirmed_identity_without_affiliation": "Identifisert uten tilhørighet",
```

brukes tilknytning andre plasser..?
```suggestion
        "confirmed_identity_without_affiliation": "Identifisert uten tilknytning",
```